### DISCUSSION TITLE
Visualizzazione Globaleaks su smartphone

### DISCUSSION THREAD

**Discussion by PTKaroveggia** [globaleaks/globaleaks-whistleblowing-software#3752] [Q&A]
Ciao a tutti, ho problemi a visualizzare Globaleaks su smartphone. La schermata che si presenta e la seguente. 
![Foto](https://github.com/globaleaks/GlobaLeaks/assets/92145511/67ef854b-eb62-4a12-9af7-509331f9e61a)

Come posso correggere tale visualizzazione? 
Grazie mille

**Comment by evilaliv3** [comment]
Grazie per la segnalazione.

In realtà non c'e' nulla che possiate fare.
Provvediamo a testare una simile situazione.

In generale la piattaforma usa tecnologie responsive che si adattano alle
dimensioni dei cellulari ma alcune interfacce per loro natura o per
configurazione potrebbero non essere del tutto fruibili.

**Reply by PTKaroveggia** [reply]
D'accordo. Grazie mille per la risposta
